generate_tuxsuite.py

```diff
@@ -68,8 +71,18 @@ def emit_tuxsuite_yml(config, tree):
             if "make_variables" in build:
                 current_build.update(
                     {"make_variables": build["make_variables"]})
-            tuxsuite_buildset["sets"][0]["builds"] += [current_build]
 
+            if "defconfig" in str(build["config"]):
```

Comment:
isn't `build["config"]` already a `str`? Why convert it again?

Reply:
Not necessarily, this could be a list.